packages/components/tour/src/content.ts

```diff
@@ -0,0 +1,53 @@
+import { buildProps, definePropType } from '@element-plus/utils'
+import type { ExtractPropTypes } from 'vue'
+import type { Placement, Strategy, VirtualElement } from '@floating-ui/dom'
+
+const tourStrategies = ['absolute', 'fixed'] as const
+
+const tourPlacements = [
+  'top-start',
+  'top-end',
+  'top',
+  'bottom-start',
+  'bottom-end',
+  'bottom',
+  'left-start',
+  'left-end',
+  'left',
+  'right-start',
+  'right-end',
+  'right',
+] as const
```

Comment:
## Suggestion

```jsx
import { placements } from '@popperjs/core'
```

Reply:
The 'placements' exported by '@popperjs/core' is different from that of '@floating-ui/dom', but '@floating-ui/dom' does not export it. so i write it in here,  just like the tooltip-V2 component.

Reply:
https://github.com/element-plus/element-plus/blob/ec34092e455823ba43693d7846de39e6f9cfaa1c/packages/components/tooltip-v2/src/content.ts#L8